common/test/keyboards/full_caps_3620_3621/source/full_caps_3620_3621.kmn

```diff
@@ -25,11 +25,10 @@ group(NewContext) readonly
     nomatch > use(nextLayer)
 
 group(PostKeystroke) readonly
-    c this is a heuristic that we can use until we support a CAPS layer (#3620):
-    c if we find two+ characters in capitals, leave us in shift layer,
-    c assuming the user hasn't just switched layer themselves. Once we have a
-    c proper CAPS layer, we'll get rid of this.
-    if(&layerChanged = "0") any(caps) any(caps) > context
+    c this is a heuristic: if we find two+ characters in capitals, leave
+    c us in shift layer, assuming the user hasn't just switched layer
+    c themselves. Not sure if this is a good idea!
+    c if(&layerChanged = "0") if(&layer = 'shift') any(caps) any(caps) > layer('caps') c ??? is this a good idea?
```

Comment:
do you want to keep this comment?

Reply:
Perhaps I should probably remove that heuristic altogether? I will leave it for now but think about it for a follow-up PR.